cloud/team/team.go

```diff
@@ -757,3 +758,202 @@ func ListTeamUsers(teamID string, out io.Writer, client astrocore.CoreClient) (e
 	fmt.Println("The selected team has no members")
 	return nil
 }
+
+func GetDeploymentTeams(client astrocore.CoreClient, deployment string, limit int) ([]astrocore.Team, error) {
+	offset := 0
+	var teams []astrocore.Team
+	ctx, err := context.GetCurrentContext()
+	if err != nil {
+		return nil, err
+	}
+
+	for {
+		includeDeploymentRoles := true
```

Comment:
nit, dont need to set this variable everytime in the for loop
prob makes sense out of the `for` loop since the value never changes

Reply:
done